docs/content/preview/deploy/manual-deployment/verify-deployment.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Verify deployment
-headerTitle: Verify deployment
+title: Verify Manual Deployment
+headerTitle: Verify Manual Deployment
```

Comment:
leave as Verify deployment.
And headerTitles, like linkTitles, are sentence case